frontend/src/types/User.ts

```diff
@@ -0,0 +1,27 @@
+import type { Chat } from "./Chat";
+import type { Game, GameRequest } from "./Game";
+import type { Relationship } from "./Relationship";
+
+interface Achievement {
+    name: string;
+}
+
+export interface User {
+    id: number;
+    name: string;
+
+    is_intra: boolean;
+    intra_id: number;
+    avatar: string;
+
+    matchmaking_request: GameRequest[];
+    games: Game[];
+
+    achievements: Achievement[];
+    relationships: Relationship[];
+
+    chats: Chat[];
+
+    created_at: Date;
+    updated_at: Date;
+}
```

Comment:
Just something I thought of now..
Would it be easy to already have an array of friends and an array of blocked?
Since we'll have to check the blocked array quite a bit for the "me" user... (you should not see that persons messages/profile anywhere)
Or maybe that's a backend thing.. not sure

Reply:
Definitely useful, I'll. add it to the profileList ticket